src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_emitter.cpp

```diff
@@ -309,7 +325,8 @@ void jit_emitter::restore_context(const std::vector<size_t>& gpr_regs,
                 continue;
             }
 
-            h->ldr(Xbyak_aarch64::QReg(reg_idx), post_ptr(h->sp, get_vec_length()));
+            const auto shift = ov::intel_cpu::rnd_up(get_vec_length(), sp_alignment);
+            h->ldr(Xbyak_aarch64::QReg(reg_idx), post_ptr(h->sp, shift));
```

Comment:
Do I understand correctly that we don't need to cast to int32 for `post_ptr`, but need for `pre_ptr` and `ptr`?